src/axom/quest/examples/shaping_driver.cpp

```diff
@@ -111,6 +119,117 @@ struct Input
 public:
   bool isVerbose() const { return m_verboseOutput; }
 
+  /// Generate an mfem Cartesian mesh, scaled to the bounding box range
+  mfem::Mesh* createBoxMesh()
```

Comment:
New inline mesh capability in shaping_driver adapted from similar capability in the `data_collection_util`

We should centralize this, e.g. in `quest/utility` so it can be used in other places in axom.